ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/results/privacy/PrivateTransactionReceiptResult.java

```diff
@@ -30,6 +32,13 @@
   "from",
   "to",
   "output",
+  "publicHash",
+  "privateHash",
+  "privateFrom",
+  "privateFor",
+  "privacyGroupId",
+  "status",
+  "revertReason",
   "logs",
 })
 public class PrivateTransactionReceiptResult {
```

Comment:
Do we need to add `@JsonInclude(Include.NON_NULL)` to ensure that `privateFor` and `privacyGroupId` won't be returned in the same response json? In the JI we say that when one is present the other won't be.